src/admin/adminuser.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export enum UserType {
+    SuperAdmin = 'Super-Admin',
+    Admin = 'Admin',
+    StandardAdmin = 'Standard-Admin',
+  }
+
+@Schema()
```

Comment:
Add  this so we can use .id instead of _id
```suggestion
@Schema({ timestamps: true, id: true })
```